packages/core/src/utils.js

```diff
@@ -368,6 +368,26 @@ export function getUiOptions(uiSchema) {
     }, {});
 }
 
+export function getDisplayLabel(schema, uiSchema, rootSchema) {
```

Comment:
@epicfaace something I thought of this morning... even though I suspect this is covered anecdotally in base tests, should I add a specific unit test for the new extracted method?

Reply:
Yeah, that would also be good, especially because it is now exported in utils.

Reply:
Check out the tests I added and let me know if you think I should add any others.

Reply:
Looks good!